packages/core/src/ComponentsManager.ts

```diff
@@ -35,6 +35,7 @@ export class ComponentsManager {
 
   // Physics
   private _colliders: DisorderedArray<Collider> = new DisorderedArray();
+  private _characterControllers: DisorderedArray<CharacterController> = new DisorderedArray();
```

Comment:
Maybe is time to use physicManager to mange phycis compoments, include `_colliders` and `_characterControllers`